File: 2.0/java/calls/getCallRecording.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.CallRecording;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        CallRecording rec = client.callsApi().getCallRecording(379506003L, "callId,campaignId,lengthInBytes,mp3Url");
    }
}

File: 2.0/csharp/calls/getCallRecording.cs
using CallfireApiClient;
using CallfireApiClient.Api.Campaigns.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        CallRecording rec = client.CallsApi.GetCallRecording(379506003, "callId,campaignId,lengthInBytes,mp3Url");
    }
}
